CHANGELOG.md

```diff
@@ -19,7 +20,8 @@
 * [Fix] Fix boxplot for duckdb native ([#728](https://github.com/ploomber/jupysql/issues/728))
 * [Fix] Fix error when using SQL Server with pyodbc that caused queries to fail due to multiple open result sets
 * [Fix] Improves performance when converting DuckDB results to `pandas.DataFrame`
-* [Feature] Add DBAPI support for `%sqlcmd profile`(#743)
+* [Fix] Fixes a bug when converting a CTE stored with `--save` into a `pandas.DataFrame` via `.DataFrame()`
+* [Doc] Add Redshift tutorial
```

Comment:
looks like a merge went bad? this entries look unrelated to your PR

Reply:
Where did you find this change? It doesn't appear on my end

Reply:
weird. I don't see it anymore. marking as fixed